### ISSUE TITLE
preserveAspectRatio="none"

### ISSUE THREAD

**Issue by achimmihca** [anup-a/svgwave#72]
Firefox only stretches an SVG freely if it has `preserveAspectRatio="none"` set on the root svg element (see [StackOverflow](https://stackoverflow.com/a/72773488/4412885))

Please add this property to the generated SVG code.

**Comment by anup-a** [comment]
Hi @achimmihca, Did you mean this https://codepen.io/anup-a/pen/OJvBebv ?

If yes, then this behaviour is consistent across browsers. But yeah I guess it would be helpful to include preserveAspectRatio="none".

**Comment by achimmihca** [comment]
Yes, this is what I mean. I expected the SVG to change size (i.e., stretch freely) according to the div I put it in. This can be achieved via `preserveAspectRatio="none"`
